drivers/storage/portworx/component/lighthouse.go

```diff
@@ -147,75 +147,79 @@ func (c *lighthouse) createServiceAccount(
-					Resources: []string{"secrets"},
-					Verbs:     []string{"get", "create", "update"},
-				},
-				{
-					APIGroups: []string{""},
-					Resources: []string{"configmaps"},
-					Verbs:     []string{"get", "create", "update"},
-				},
-				{
-					APIGroups: []string{""},
-					Resources: []string{"nodes"},
-					Verbs:     []string{"get", "list", "watch"},
-				},
-				{
-					APIGroups: []string{""},
-					Resources: []string{"services"},
-					Verbs:     []string{"get", "list", "watch", "create"},
-				},
-				{
-					APIGroups: []string{"stork.libopenstorage.org"},
-					Resources: []string{"*"},
-					Verbs:     []string{"get", "list", "create", "delete", "update"},
-				},
-				{
-					APIGroups: []string{"monitoring.coreos.com"},
-					Resources: []string{
-						"alertmanagers",
-						"prometheuses",
-						"prometheuses/finalizers",
-						"servicemonitors",
-						"prometheusrules",
-					},
-					Verbs: []string{"*"},
-				},
-				{
-					APIGroups:     []string{"security.openshift.io"},
-					Resources:     []string{"securitycontextconstraints"},
-					ResourceNames: []string{PxRestrictedSCCName, "anyuid"},
-					Verbs:         []string{"use"},
-				},
-				{
-					APIGroups:     []string{"policy"},
-					Resources:     []string{"podsecuritypolicies"},
-					ResourceNames: []string{constants.PrivilegedPSPName},
-					Verbs:         []string{"use"},
+			{
+				APIGroups: []string{"extensions", "apps"},
+				Resources: []string{"deployments"},
+				Verbs:     []string{"get", "list"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"secrets"},
+				Verbs:     []string{"get", "create", "update"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"configmaps"},
+				Verbs:     []string{"get", "create", "update"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"nodes"},
+				Verbs:     []string{"get", "list", "watch"},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"services"},
+				Verbs:     []string{"get", "list", "watch", "create"},
+			},
+			{
+				APIGroups: []string{"stork.libopenstorage.org"},
+				Resources: []string{"*"},
+				Verbs:     []string{"get", "list", "create", "delete", "update"},
+			},
+			{
+				APIGroups: []string{"monitoring.coreos.com"},
+				Resources: []string{
+					"alertmanagers",
+					"prometheuses",
+					"prometheuses/finalizers",
+					"servicemonitors",
+					"prometheusrules",
 				},
+				Verbs: []string{"*"},
+			},
+			{
+				APIGroups:     []string{"security.openshift.io"},
+				Resources:     []string{"securitycontextconstraints"},
+				ResourceNames: []string{sccName, "anyuid"},
+				Verbs:         []string{"use"},
+			},
+			{
+				APIGroups:     []string{"policy"},
+				Resources:     []string{"podsecuritypolicies"},
+				ResourceNames: []string{constants.PrivilegedPSPName},
+				Verbs:         []string{"use"},
 			},
 		},
-	)
+	}
+
+	return k8sutil.CreateOrUpdateClusterRole(c.k8sClient, clusterRole)
```

Comment:
This seems to be a new addition... was this a mistake, or was this line required ?

Reply:
This method was returning **k8sutil.CreateOrUpdateClusterRole** , changed it to last line to add conditional statement of SCC name before creating the cluster role.